### DISCUSSION TITLE
Potential Issue with Teensy LC

### DISCUSSION THREAD

**Discussion by cloud1242** [Makuna/NeoPixelBus#408] [General]
I am sure there is a better way to raise this issue but, I believe I have encountered a compatibility issue with the Teensy LC (the error only shows if I compile for LC at 48 Mhz, Uno is fine). Here is the resulting error:

```
In file included from C:\Users\15857\Documents\Arduino\libraries\NeoPixelBus_by_Makuna\src/NeoPixelBus.h:106:0,
                 from C:\Users\15857\Desktop\Arduino\jiriSphere\jiriSphere.ino:1:
C:\Users\15857\Documents\Arduino\libraries\NeoPixelBus_by_Makuna\src/internal/NeoArmMethod.h:803:1: error: expected initializer before 'typedef'
 typedef NeoArmWs2812xMethod NeoWs2813Method;
 ^
C:\Users\15857\Documents\Arduino\libraries\NeoPixelBus_by_Makuna\src/internal/NeoArmMethod.h:809:9: error: 'NeoArmApa106Method' does not name a type
 typedef NeoArmApa106Method NeoApa106Method;
         ^
Error compiling for board Teensy LC.

```

Hope this isn't a waste of time!

***Edit: I wrote 46 Mhz, this should have been 48 Mhz.

**Comment by Makuna** [comment]
@cloud1242 Could you look in the lines above this error for one that has the defines on the line, like -Dblahblah; they tend to be the longest line and include many defines. specifically -DF_CPU=* and -D__MKL26Z64__.

With the Teensy LC, only 48Mhz is supported, it should give a compile error stating this if you set it to anything else.

**Comment by Makuna** [comment]
edit the file "NeoArmMethod.h", goto line 337, at the end add a semicolon like
```
typedef NeoArm800KbpsMethod NeoArmApa106Method;
``` 

I wish the team who supports the Teensy Arduino boards would make their board support installable through the Boards Manager.  Its such a pain to install and keep upto date for those of us who want to support them but don't actively use it.

**Comment by Makuna** [comment]
https://github.com/Makuna/NeoPixelBus/pull/409

**Comment by cloud1242** [comment]
Hey Makuna, 

Thank you for the swift reply. I helped me to track the line ending down when I simply could not on my own. 

Cheers!
